ledger/participant-integration-api/src/test/suite/scala/platform/store/dao/JdbcLedgerDaoH2DatabaseSpec.scala

```diff
@@ -12,12 +12,12 @@ final class JdbcLedgerDaoH2DatabaseSpec
     with Matchers
     with JdbcLedgerDaoSuite
     with JdbcLedgerDaoBackendH2Database
+    with JdbcLedgerDaoPackagesSpec
```

Comment:
Note: moving the package spec to the top, so that all other specs can access the stored DAML-LF packages for the retrofitting of types.